### PR TITLE
small cleanup fixes

### PR DESCRIPTION
**Pull request by jafu888**
These are the small changes we talked about.

### CODE REVIEW

**Review by jswong65 — COMMENTED** [review]
One minor issue

File: constraintlayout/core/src/main/java/androidx/constraintlayout/core/widgets/analyzer/Grouping.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 
     private static final boolean DEBUG = false;
     private static final boolean DEBUG_GROUPING = false;
+    private static final boolean DO_NOT_USE = false;
```
[the remark this answers is not in the document]
Comment (jafu888) [reply constraintlayout/core/src/main/java/androidx/constraintlayout/core/widgets/analyzer/Grouping.java:45]: Played around with many names 
currently the code says if (DO_NOT_USE  && .... ) {
this feels clear 
if (FORCE_USE && ...) {

} 

would seem like the section is being used.

**Comment by jafu888** [comment]
DO NOT SUBMIT SOME MISTAKES!

File: constraintlayout/core/src/main/java/androidx/constraintlayout/core/widgets/analyzer/Grouping.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jafu888) [reply constraintlayout/core/src/main/java/androidx/constraintlayout/core/widgets/analyzer/Grouping.java:45]: Good catch/ Unclear comment
it should be ::
private static final boolean FORCE_USE = true;

**Pull request merged**